src/components/loading-page/loading.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from "react";
+import { ILoadingProps } from "./loading.interface";
+import { StylesWrapper } from "./loading.styles";
+
+export const Loading: FC<ILoadingProps> = () => {
+  return (
+    <StylesWrapper>
+      <div className="spinner-container">
+        <div className="loading-spinner"></div>
+      </div>
+      <p>Loading...</p>
```

Comment:
🦆 You can add this to constants

Reply:
🥔 I've updated and used constant for the text